src/lbh/components/lbh-tag/_tag.scss

```diff
@@ -0,0 +1,18 @@
+@import "node_modules/govuk-frontend/govuk/components/tag/tag";
```

Comment:
This needs to also import all the mixins, etc.

Reply:
Good spot. Also please import this tag.scss file in the components' `all.scss`